docs/index.html

```diff
@@ -0,0 +1,4 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Toolkit for Bare Metal Services</title>
```

Comment:
Sorry I didn't notice this last time (I didn't get enough sleep last night): "Services" should be "Solution"

Reply:
Updated :-)